.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - deploy
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build app
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.5
```

Comment:
do we want to bump our python version 3.5 is now EOL. This is probably a separate issue.

Reply:
Another a great question. Opened https://github.com/datamade/scrapers-us-municipal/issues/65.

Reply:
(My answer is not yet because the OCD scrapers on the server are still using Python 3.5.)